### DISCUSSION TITLE
Custom User Profile Fields

### DISCUSSION THREAD

**Discussion by choseh** [pretix/pretix#2455] [Ideas] [answered]
Hey,

It would be nice to be able to define custom fields in the user model, that allow prefilling of certain questions during Ticket orders.

Thanks,
Alex

**Comment by raphaelm** [comment, marked as answer]
If you fill out questions during checkout, you can save them to your customer profile and re-use them later.

I see no point in adding complex configuration for this?

**Reply by choseh** [reply]
thanks, another lesson learned :)

**Reply by wiffbi** [reply]
we even try to match them accross different events and similar questions :)
